src/inmanta/db/versions/v202229060.py

```diff
@@ -0,0 +1,32 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+DISABLED = False
```

Comment:
```suggestion
```

I didn't think we included this anymore in the latest versions. It's redundant since our new versioning scheme and it just adds clutter. If we take it out of this one, it will probably persist (since the next one will likely be copied from this).